src/eq.js

```diff
@@ -0,0 +1,157 @@
+   * @param {Number} [_eqsize] Constructor will accept 3 or 8, defaults to 3
+   * @return {Object} p5.EQ object
+   *
+   */
+  p5.EQ = function(_eqsize) {
+    Effect.call(this);
+
+    //p5.EQ can be of size (3) or (8), defaults to 3
+    _eqsize = _eqsize === 3 || _eqsize === 8 ? _eqsize : 3;
+
+    var factor;
+    _eqsize == 3 ? factor = Math.pow(2,3) : factor = 2;
+
+    /**
+      *  The p5.EQ is built with abstracted p5.Filter objects. 
+      *  To modify any bands, use methods of the p5.Filter API.
+      *  To add or remove bands, use p5.EQ.addBand() and p5.EQ.removeBand().
+      *  Bands are stored in an array, with indices 0 - 3, or 0 - 7
+      *  @property {Array}  bands
+      *
+    */
+    this.bands = [];
+
+    
+    var freq, res;
+    for (var i = 0; i < _eqsize; i++) {
+      if (i === _eqsize - 1) {
+        freq = 21000; 
+        res = .01;
+      } else if (i === 0) {
+        freq = 100;
+        res = .1;
+      } 
+      else if (i===1) {
+        freq = _eqsize === 3 ? 360 * factor : 360;
+        res = 1;
+      }else {
+        freq = this.bands[i-1].freq() * factor;
+        res = 1;
+      } 
+      this.bands[i] = this._newBand(freq, res);
+      if (i>0) {
+        this.bands[i-1].biquad.connect(this.bands[i].biquad);
+      } else {
+        this.input.connect(this.bands[i].biquad);
+      }
+    }
+
+    this.bands[_eqsize-1].biquad.connect(this.output);
+  };
+  p5.EQ.prototype = Object.create(Effect.prototype);
+
+  /**
+   * Process an input by connecting it to the EQ
+   * @method  process
+   * @param  {Object} src Audio source
+   */
+  p5.EQ.prototype.process = function (src) {
+    src.connect(this.input);
+  };
+
+  /** 
+   * Set the frequency and gain of each band in the EQ. This method should be
+   * called with 3 or 8 frequency and gain pairs, depending on the size of the EQ. 
+   * ex. eq.set(freq0, gain0, freq1, gain1, freq2, gain2);
+   *
+   * @method  set
+   * @param {Number} [freq0] Frequency value for band with index 0
+   * @param {Number} [gain0] Gain value for band with index 0
+   * @param {Number} [freq1] Frequency value for band with index 1
+   * @param {Number} [gain1] Gain value for band with index 1
+   * @param {Number} [freq2] Frequency value for band with index 2
+   * @param {Number} [gain2] Gain value for band with index 2
+   * @param {Number} [freq3] Frequency value for band with index 3
+   * @param {Number} [gain3] Gain value for band with index 3
+   * @param {Number} [freq4] Frequency value for band with index 4
+   * @param {Number} [gain4] Gain value for band with index 4
+   * @param {Number} [freq5] Frequency value for band with index 5
+   * @param {Number} [gain5] Gain value for band with index 5
+   * @param {Number} [freq6] Frequency value for band with index 6
+   * @param {Number} [gain6] Gain value for band with index 6
+   * @param {Number} [freq7] Frequency value for band with index 7
+   * @param {Number} [gain7] Gain value for band with index 7
```

Comment:
16 parameters is a bit unwieldy, it seems like it would be hard to keep track of which value goes where